library.properties

```diff
@@ -1,10 +1,9 @@
 name=Adafruit INA260 Library
-version=1.3.5
+version=1.3.2
```

Comment:
Looks like an old version; This should be bumped to 1.4.0 and the dependencies should remain